### PR TITLE
Update example gerald-pr.yml workflow to use Khan/actions

### PR DESCRIPTION
**Pull request by jaredly**
## Summary:
Issue: https://khanacademy.atlassian.net/browse/FEI-4582

## Test plan:

### CODE REVIEW

**Review by lillialexis — APPROVED** [review]
**M E T A**
**E**
**T**
**A**

🤯 🤯 🤯 🤯

File: setup-files/gerald-pr.yml [hunk 1]
```diff
@@ -1,21 +1,17 @@
 name: Gerald - Notify and Request Reviewers On Pull Request
 'on':
- - pull_request
+  pull_request:
+    # ready_for_review isn't included by default, so we add it
+    # here to have this workflow re-run when a pr is converted
+    # from "draft" to "not draft".
+    # We also add "edited" to re-check when the base branch
+    # is changed.
+    types: [opened, synchronize, reopened, ready_for_review, edited]
```
Comment (somewhatabstract) [inline setup-files/gerald-pr.yml:9]: ASIDE: I wish edited could be "base branch changed" and then title edits wouldn't cause a rerun. I don't think they have a trigger that specific though. No change requested.

File: setup-files/gerald-pr.yml [hunk 2]
```diff
@@ -1,21 +1,17 @@
...
 jobs:
   gerald:
     runs-on: ubuntu-latest
     steps:
-      - name: Check out base branch
-        uses: actions/checkout@v2
+      - uses: Khan/actions@gerald-pr-v0
```
Comment (somewhatabstract) [inline setup-files/gerald-pr.yml:14]: Nice

**Review by somewhatabstract — APPROVED** [review]
Thanks for updating this!

File: setup-files/gerald-pr.yml (hunk 1 above)
Comment (jaredly) [reply setup-files/gerald-pr.yml:9]: yup yup.
looks like we ought to be able to determine that it was a title change and then bail though
image

Comment (somewhatabstract) [reply setup-files/gerald-pr.yml:9]: Oh great. Something to consider for future updates.

**Pull request merged**